conf/test_failed.config

```diff
@@ -35,7 +35,6 @@ params {
     ignore_failed_trimming = true
     ignore_empty_input_files = true
     ignore_failed_filtering = true
-    ancombc = true
```

Comment:
Is this on purpose?

Reply:
Yes, because for an unknown reason that makes the test fail. That param is also activated (and succeeding) in profile test, so I am not overly concerned about it fr now.